tracee-ebpf/Makefile

```diff
@@ -23,7 +23,7 @@ BPF_SRC := tracee/tracee.bpf.c
 ifndef CORE
 OUT_BPF := $(OUT_DIR)/tracee.bpf.$(subst .,_,$(KERN_RELEASE)).$(subst .,_,$(VERSION)).o
 else
-OUT_BPF := $(OUT_DIR)/tracee.bpf.core.$(subst .,_,$(VERSION)).o
```

Comment:
Why did you remove tracee version from the file?
Tracee binary version should match the bpf code version

Reply:
For the embed directive I can't pass the build time variable into the actual directive comment, so it'd be impossible to find it. Plus if it's being embedded anyway, why does it need the version?

Reply:
SGTM